webroot/adm/keysets.html

```diff
@@ -0,0 +1,103 @@
+<html>
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="/js/main.js"></script>
+</head>
+<body>
+<h1>UID2 Admin - Keyset Access Management (Administrator)</h1>
+
+<a href="/">Back</a>
+
+<br>
+<br>
+
+<h3>Inputs</h3>
+
+<label for="siteId">Site Id:</label>
```

Comment:
Same as above

Reply:
Need to have both siteId and keysetId. The endpoint /api/sharing/keyset is consuming the site id from the request payload.